nvflare/app_common/widgets/validation_json_generator.py

```diff
@@ -64,11 +64,22 @@ def handle_event(self, event_type: str, fl_ctx: FLContext):
                         if data_client not in self._val_results:
                             self._val_results[data_client] = {}
                         self._val_results[data_client][model_owner] = dxo.data
+                    elif dxo.data_kind == DataKind.COLLECTION:
```

Comment:
Collection could be a list or a dict. This code here only works for dict. This should change to use get_leaf_dxos(), as in other places.

Reply:
Good point. Changed to use get_leaf_dxos.